src/components/divider/divider.ts

```diff
@@ -0,0 +1,79 @@
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { themes } from '../../theming/theming-decorator.js';
+import { registerComponent } from '../common/definitions/register.js';
+import { styles } from './themes/divider.base.css.js';
+import { styles as shared } from './themes/shared/divider.common.css.js';
+import { all } from './themes/themes.js';
+
+/**
+ * The igc-divider allows the content author to easily create a horizontal/vertical rule as a break between content to better organize information on a page.
+ *
+ * @element igc-divider
+ *
+ */
+
+@themes(all)
+export default class IgcDividerComponent extends LitElement {
+  public static readonly tagName = 'igc-divider';
+  public static override styles = [styles, shared];
+
+  private _internals: ElementInternals;
+
+  /* blazorSuppress */
+  public static register() {
+    registerComponent(IgcDividerComponent);
+  }
+
+  private _vertical = false;
+
+  /**
+   * Whether to render a vertical divider line.
+   * @attr
+   */
+  public set vertical(value: boolean) {
```

Comment:
```suggestion
  /**
   * Whether to render a vertical divider line.
   * @attr
   */
  @property({ type: Boolean, reflect: true })
  public set vertical(value: boolean) {
```

You've dropped the property decorator. Add it back to fix the failing unit test.